feather/common/src/events.rs

```diff
@@ -58,3 +58,6 @@ pub struct ChunkLoadEvent {
 pub struct ChunkLoadFailEvent {
     pub position: ChunkPosition,
 }
+
+#[derive(Debug)]
+pub struct TablistExtrasUpdateEvent;
```

Comment:
#522 is now merged, so quill-compatible events should be in `quill/common/srs/events/change.rs`, implement Serialize, Deserialize, Clone, be defined in `quill/common/src/component.rs` in `HostComponent` enum and have `bincode_component_impl!` in the end.